climada/hazard/storm_europe.py

```diff
@@ -159,27 +183,30 @@ def from_footprints(cls, path, description=None, ref_raster=None, centroids=None
 
         LOGGER.info('Commencing to iterate over netCDF files.')
 
-        haz = cls()
-        for file_name in file_names:
-            if any(fo in file_name for fo in files_omit):
-                LOGGER.info("Omitting file %s", file_name)
-                continue
-            new_haz = cls._read_one_nc(file_name, centroids, intensity_thres)
-            if new_haz is not None:
-                haz.append(new_haz)
-
+        files_to_read = [file for file in file_names if file not in files_omit]
+        LOGGER.info(
+            "Omitting files %s", [
+                file for file in file_names if file in files_omit])
```

Comment:
```suggestion
        if any(fo in file_name for fo in files_omit):
            LOGGER.info("Omitting files %s", [
                file for file in file_names if file in files_omit])
```

Reply:
Very important suggestion, thank you! But I did not want to add yet another list comprehension. I found a different solution using sets, see 6f0acc1ce498a59090fd8669f68eda50a928cb44

Reply:
Rather very unimportant, right? But yes, sets make more sense. 👍 (I was thinking along the same lines. Let's just hope the list was rather a bug than e feature, i.e., providing a path list with duplicates is not an intended valid use case. 🤞)